app/app.go

```diff
@@ -52,8 +57,13 @@ func (a *App) Setup() {
 		a.initCPUProfiling()
 	}
 
-	// initialize enterprise features, if available
-	a.initEnterpriseFeatures()
+	if a.options.EnterpriseToken == "" {
```

Comment:
Why do we need a token (string) instead of a boolean?

Reply:
Because we want control-plane service to verify that the token is valid. This will happen at a second stage.